### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by grouma**

### CODE REVIEW

File: frontend_server_client/mono_pkg.yaml [hunk 1]
```diff
@@ -4,4 +4,4 @@ stages:
     - group:
       - dartfmt
       - dartanalyzer: --fatal-infos --fatal-warnings .
-      dart: 2.9.0-19.0.dev
+      dart: 2.10.0
```
Comment (kevmoo) [inline frontend_server_client/mono_pkg.yaml:7]: Is this the lower-bound on the package, too?

Good to keep those in sync!

File: mono_repo.yaml [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
-    chrome: stable
-  before_install:
-    - tool/travis_setup.sh
-    - export PATH=$PATH:$PWD
-  after_failure:
-    - tool/report_failure.sh
-  services:
-    - xvfb
+github:
+  cron:
```
Comment (kevmoo) [inline mono_repo.yaml:4]: Need to actually put a value here!

File: tool/ci.sh [hunk 3]
```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
```
Comment (kevmoo) [inline tool/ci.sh:1]: Did you delete the `tool/travish.sh` file, too?

**Review by kevmoo — APPROVED** [review]
Remove the WIP from the PR, too 😄

**Comment by grouma** [comment]
> Remove the WIP from the PR, too 😄

Looks like we have a lot of flakes so I'll be hacking on this for a while. Thanks for the initial review. I'll ping you when it's ready for a final look.

**Comment by kevmoo** [comment]
@grouma – we have the ability to do the notifications, too – don't forget that!

File: tool/ci.sh (hunk 3 above)
Comment (kevmoo) [reply tool/ci.sh:1]: you did!

**Comment by grouma** [comment]
Merging on the flake.

**Pull request merged**